app/controllers/application_controller.rb

```diff
@@ -1,6 +1,10 @@
 class ApplicationController < ActionController::Base
   layout :layout_by_resource
 
+  def check
+    render json: { status: "OK", version: ENV["SHA"], environment: Rails.env }, status: :ok
+  end
```

Comment:
Does the json include the Rails environment ?  Is there anything in there that would be useful to bad people ?

Reply:
looks like a simple json response object: status string, encrypted version string (SHA), string saying the environment: "production" or "development"